contracts/deploy/055_curve_amo.js

```diff
@@ -0,0 +1,478 @@
+      signature:
+        "setRewardTokenConfig(address,uint16,uint16,address,uint256,bool)",
+      args: [
+        assetAddresses.CVX,
+        300,
+        100,
+        assetAddresses.sushiswapRouter,
+        MAX_UINT256,
+        true,
+      ],
+    },
+  ];
+};
+
+const deployHarvesterAndOracleRouter = async ({
+  deployWithConfirmation,
+  withConfirmation,
+  ethers,
+}) => {
+  await deployWithConfirmation("OETHOracleRouter");
+
+  const { deployerAddr, governorAddr } = await getNamedAccounts();
+  const sDeployer = await ethers.provider.getSigner(deployerAddr);
+  const dHarvesterProxy = await deployWithConfirmation("OETHHarvesterProxy");
+  const cVaultProxy = await ethers.getContract("OETHVaultProxy");
+  const cVault = await ethers.getContractAt("OETHVault", cVaultProxy.address);
+  const cOETHOracleRouter = await ethers.getContract("OETHOracleRouter");
+  console.log(`Harvester proxy deployed at: ${dHarvesterProxy.address}`);
+
+  const cHarvesterProxy = await ethers.getContractAt(
+    "OETHHarvesterProxy",
+    dHarvesterProxy.address
+  );
+
+  const dHarvester = await deployWithConfirmation("OETHHarvester", [
+    cVaultProxy.address,
+  ]);
+
+  await withConfirmation(
+    cHarvesterProxy.connect(sDeployer)[
+      // eslint-disable-next-line
+      "initialize(address,address,bytes)"
+    ](dHarvester.address, deployerAddr, [])
+  );
+
+  const cHarvester = await ethers.getContractAt(
+    "OETHHarvester",
+    cHarvesterProxy.address
+  );
+
+  await withConfirmation(
+    // CRV/ETH
+    cOETHOracleRouter.cacheDecimals(addresses.mainnet.CRV)
+  );
+
+  await withConfirmation(
+    // CVX/ETH
+    cOETHOracleRouter.cacheDecimals(addresses.mainnet.CVX)
+  );
+
+  await withConfirmation(
+    // rETH/ETH
+    cOETHOracleRouter.cacheDecimals(addresses.mainnet.rETH)
+  );
+
+  await withConfirmation(
+    // stETH/ETH
+    cOETHOracleRouter.cacheDecimals(addresses.mainnet.stETH)
+  );
+
+  await withConfirmation(
+    cHarvester.connect(sDeployer).transferGovernance(guardianAddr)
+  );
+
+  // Some of the harvester governance actions are executed when deploying Curve
+  // strategy
+  return {
+    actions: [
+      {
+        contract: cHarvester,
+        signature: "claimGovernance()",
+        args: [],
+      },
+      {
+        contract: cVault,
+        signature: "setPriceProvider(address)",
+        args: [cOETHOracleRouter.address],
+      },
+    ],
+    cHarvester,
+  };
+};
+
+// const reDeployOETH = async ({
+//   deployWithConfirmation,
+//   withConfirmation,
+//   ethers,
+// }) => {
+//   const cOETHProxy = await ethers.getContract("OETHProxy");
+//   const dOETH = await deployWithConfirmation("OETH");
+//
+//   return [
+//     {
+//       // Upgrade OETH proxy
+//       contract: cOETHProxy,
+//       signature: "upgradeTo(address)",
+//       args: [dOETH.address],
+//     }
+//   ];
+// };
+
+const deployCurve = async ({
```

Comment:
Gotta make sure this is turned off for the live deploy

Reply:
yup yup thanks for flagging